lib/DDG/Spice/Piratetalk.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Piratetalk;
+
+# ABSTRACT: Piratetalk will translate texts into Pirate language when triggered with the trigger words mentioned below. 
+# For example, "Pirate Speak Hello" will return "Ahoy!" 
+
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; 
+spice wrap_jsonp_callback => 1; 
+
+spice to => 'https://pirate-speak.herokuapp.com/piratespeak?text=$1';
```

Comment:
@thejasbabu is this your API or do you know who the maintainer is?

Do you know if this is using a free Heroku plan? If so, I'm worried about the required sleep time for the server, which I imagine will cause the response times to be slow.